src/_pytest/doctest.py

```diff
@@ -536,6 +537,18 @@ def _find(
                         tests, obj, name, module, source_lines, globs, seen
                     )
 
+            def _from_module(self, module, object):
```

Comment:
Can we wrap this in `if sys.version_info < (3, 13):`?

Reply:
Great call 👍